src/tooltip/tooltip.js

```diff
@@ -260,6 +260,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
              * Observe the relevant attributes.
              */
             attrs.$observe( type, function ( val ) {
+              if (val && val.length >= 1 && val.charAt(0) == '@') {
```

Comment:
<pre>
if (val && val.charAt(0) === '@') {
</pre>
is sufficient.  `String.prototype.charAt` will return an empty string if val is an empty string.